src/RestApi/StoreApi/Routes/CartAddItem.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Cart add item route.
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\RestApi\StoreApi\Routes;
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\WooCommerce\Blocks\RestApi\StoreApi\Utilities\CartController;
+
+/**
+ * CartAddItem class.
+ */
+class CartAddItem extends AbstractRoute {
+	/**
+	 * Get the namespace for this route.
+	 *
+	 * @return string
+	 */
+	public function get_namespace() {
+		return 'wc/store';
+	}
+
+	/**
+	 * Get the path of this REST route.
+	 *
+	 * @return string
+	 */
+	public function get_path() {
+		return '/cart/add-item';
```

Comment:
Do we need to add this to the API docs? I'm getting a bit lost - there are a few routes related to cart items now.

Reply:
All endpoints need docs when we're done, during cooldown.